src/CqlBuilderPanel/definitionsSection/definitions/Definitions.tsx

```diff
@@ -110,38 +111,41 @@ const Definitions = ({
         header: "",
         accessorKey: "apply",
         cell: (row: any) => {
-          if (!canEdit) {
-            return null;
-          }
           return (
             <Stack
               direction="row"
               alignItems="center"
               data-testid="definition-actions"
             >
+              {canEdit && (
+                <ToolTippedIcon
+                  tooltipMessage="Delete"
+                  buttonProps={{
+                    "data-testid": `delete-button-${row.cell.row.id}`,
+                    "aria-label": `delete-button-${row.cell.row.id}`,
+                    size: "small",
+                    onClick: (e) => {
+                      setSelectedDefinition(row.row.original.name);
+                      if (!isCQLUnchanged) {
+                        setDiscardDialog({ open: true, operation: "delete" });
```

Comment:
do you think it is possible to improve the code coverage for user actions?

Reply:
added a not.tobeindocument check